### PR TITLE
Optimize joern

### PR DESCRIPTION
**Pull request by maximzubkov**

### CODE REVIEW

File: dataset/classification_datasets/graph_dataset.py [hunk 1]
```diff
@@ -56,10 +56,14 @@ def process(self):
             e = json.loads(graph["edges"])
             v = json.loads(graph["vertexes"])
 
-            x = torch.LongTensor([self.v_type2id[v_["label"]] for v_ in v])
```
Comment (SpirinEgor) [inline dataset/classification_datasets/graph_dataset.py:59]: To avoid multiple calling for the UNKNOWN label:
```
unk_id = self.v_type2id["UNKNOWN"]
x = torch.tensor([self.v_type2id.get(v_["label"], unk_id) for v_ in v], dtype=torch.long)
```

File: dataset/classification_datasets/graph_dataset.py [hunk 2]
```diff
@@ -56,10 +56,14 @@ def process(self):
...
+            x = torch.LongTensor([
+                self.v_type2id[v_["label"]] if v_["label"] in self.v_type2id else self.v_type2id["UNKNOWN"] for v_ in v
+            ])
             y = dirname(file)
             edge_index = torch.stack([torch.LongTensor([e_["in"], e_["out"]]) for e_ in e], dim=-1)
-            edge_attr = torch.LongTensor([self.e_type2id[e_["label"]] for e_ in e])
+            edge_attr = torch.LongTensor([
+                self.e_type2id[e_["label"]] if e_["label"] in self.e_type2id else self.e_type2id["UNKNOWN"] for e_ in e
+            ])
```
Comment (SpirinEgor) [inline dataset/classification_datasets/graph_dataset.py:66]: And same here

**Review by SpirinEgor — APPROVED** [review]
